ibis/expr/tests/test_table.py

```diff
@@ -1288,7 +1289,7 @@ def g(x):
 def test_pickle_table_expr():
     schema = [('time', 'timestamp'), ('key', 'string'), ('value', 'double')]
     t0 = ibis.table(schema, name='t0')
-    raw = pickle.dumps(t0)
+    raw = pickle.dumps(t0, protocol=2)
```

Comment:
Why are you passing a specific protocol version here?

Reply:
On py27 serializeing object with `__slots__` requires protocol 2.